src/http/ngx_http_core_module.c

```diff
@@ -4049,6 +4049,13 @@ ngx_http_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
         }
 #endif
 
+#if (NGX_LINUX)
+        if (ngx_strcmp(value[n].data, "mptcp") == 0) {
+            lsopt.protocol = IPPROTO_MPTCP;
```

Comment:
mmh, I guess we will need:

```
#ifndef IPPROTO_MPTCP
#define IPPROTO_MPTCP 262
#endif
```

not to have compilation errors on older platforms. But not sure where it would be best to do that.

Reply:
I believe it is required in each of the modules (`http`, `mail`, `stream`) as they can be compiled independently of each other.

Reply:
> Yes, but this could be set in a common header (I didn't really check, maybe something like `ngx_socket.h`)?

(was it not possible to do ↑?)